docs/memory-server.md

```diff
@@ -0,0 +1,117 @@
+then Metrictank will need to fallback to Cassandra more often.
+
+## Configuration guidelines
+
+See [the example config](https://github.com/raintank/metrictank/blob/master/metrictank-sample.ini) for an overview and basic explanation of what the config values are.
+Some of the values related to chunking and compression are a bit harder to tune, so this section will explain in more detail.
+
+
+## Chunk sizing and num chunks to keep in memory
+
+### Basic guideline
+
+`chunkspan` is how long of a timeframe should be covered by your chunks. E.g. you could store anywhere between 1 second to 24 hours worth of data in a chunk.
+`numchunks` is simply up to how many chunks should be retained in the ring buffers per metric. Queries for data not in the ringbuffer will hit the chunk-cache-fronted cassandra store.
+
+These two values are defined globally for raw metrics, and also within the `agg-settings` configuration, as each rollup definition needs appropriate chunk span and numchunks based on the interval of the rollup.  The settings currently apply to all metrics, we don't support finetuning on a per-metric or per-category level yet.
+
+#### Valid chunk spans
+
+Chunkspans can be set to one of 1sec, 5sec, 10sec, 15sec, 20sec, 30sec, 60sec, 90sec, 2min, 3min, 5min, 10min, 15min, 20min, 30min, 45min, 1h, 90min, 2h, 150min, 3h, 4h, 5h, 6h, 7h, 8h, 9h, 10h, 12h, 15h, 18h, 24h.  
+This list can be extended in the future.
+
+### Chunkspan
+
+The standard recommendation is at least 120 points per chunk.
+
+#### Compression efficiency
+
+The more points are contained within a chunk, the more efficiently the compression can work. This is very noticeable
+until about 120 points per chunk, at which point the improvement becomes less relevant.
+For more details, see the [go-tsz eval program](https://github.com/dgryski/go-tsz/tree/master/eval) or the 
+[results table](https://raw.githubusercontent.com/dgryski/go-tsz/master/eval/eval-results.png)
+
+#### Cassandra load
+
+Longer chunks with more points mean a lower rate of chunksaves, meaning fewer write commands and iops for cassandra.
+There is probably an upper limit where cassandra becomes unhappy with large chunksizes, but we haven't seen that yet. 
+
+#### Memory overhead
+
+Chunks come with a memory overhead (for the internal datastructure and bookkeeping).  Longer chunks with more data reduce the memory overhead.  Though this rarely seems to be a real concern.
+Longer chunks however do force you to keep more data in RAM. Let's say you want to keep 1h worth of data in RAM. With a chunksize of 1h you need to keep 2 chunks in RAM 
+(because the current chunk becomes empty at every start of a 1h interval), which in worst case leads up to 2h worth of data.  With a chunksize of 1min you need to keep 61 chunks in RAM,
+worst case being 61 minutes.
+
+#### Warmup and becoming ready for promotion to primary
+
+longer chunk sizes means a longer backfill of more older data (e.g. with kafka oldest offset),
```

Comment:
the `l` in `longer` should be uppercase because it's the beginning of a sentence